scripts/get_version.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Get VERSION from settings.py
+grep -Eo -m 1 'VERSION\s+= .*(\d+)\.\d+\.\d+.*' main/settings.py |
+	head -1 |
+	grep -Eo "\d+\.\d+\.\d+"
```

Comment:
If settings.py is canonical, it's ok to extract it from there (by any means!). I agree that 3 is nicer. It would be useful to also write this to a version.txt file for the front end build as we do with:

```
- name: Write commit SHA to file
  run: echo $GITHUB_SHA > frontends/mit-open/build/static/hash.txt
```